packages/ember-glimmer/lib/environment.js

```diff
@@ -133,6 +137,18 @@ export default class Environment extends GlimmerEnvironment {
           wrapClassAttribute(args);
           return new CurlyComponentSyntax({ args, definition, templates });
         }
+      } else {
+        // Check if it's a keyword
+        let mappedKey = keywords[key];
+        if (mappedKey) {
+          if (mappedKey !== key) {
+            path = path.map((segment) => segment === key ? mappedKey : segment);
```

Comment:
Sounds good, but lets extract it out and make it generic if possible. I have a feeling we will need this for `{{input}}` as well.